src/main/scala/net/ripe/rpki/nro/service/Ports.scala

```diff
@@ -199,15 +193,20 @@ object Ports extends Logging {
 
 
 
-  def writeRecords(records: Records, outputFile: String = s"$resultFileName", header: Boolean = true): Unit = {
-      writeResult(records.asn, records.ipv4, records.ipv6, outputFile, header)
+  def writeRecords(records: Records, outputFile: String = s"$resultFileName", header: Boolean = true, disclaimer: Boolean = false): Unit = {
```

Comment:
Prefer composition over passing around booleans for special cases

Reply:
Refactored using composition.